python/PyQt6/core/auto_generated/mesh/qgsmeshadvancedediting.sip.in

```diff
@@ -146,13 +146,31 @@ The mesh layer must be in edit mode.
 
    this method not apply new vertices to the mesh layer but only store the calculated transformation
    that can be apply later with :py:func:`QgsMeshEditor.advancedEdit()`
```

Comment:
I don't know if this is public API, but anyway, could you update these sentences while at it, please? Thanks.
```
   This method does not apply new vertices to the mesh layer but only stores the calculated transformation
   that can be applied later with :py:func:`QgsMeshEditor.advancedEdit()`
```